_topic_map.yml

```diff
@@ -2481,6 +2481,8 @@ Topics:
     File: customizing-installation-ossm
   - Name: Performance and scalability
     File: ossm-performance-scalability
+  - Name: Troubleshooting installation
```

Comment:
Is the troubleshooting documentation only going to be focused on installation? I thought this was the start of a troubleshooting assembly that we will add to over time.

I really think that the troubleshooting assembly should be second to last.